chromadb/api/segment.py

```diff
@@ -289,6 +310,16 @@ def _update(
             records_to_submit.append(r)
         self._producer.submit_embeddings(coll["topic"], records_to_submit)
 
+        self._telemetry_client.capture(
+            CollectionUpdateEvent(
+                collection_uuid=str(collection_id),
+                update_amount=len(ids),
+                with_embeddings=embeddings is not None,
```

Comment:
Why is this a boolean in some places and an int in others?

Reply:
Oh, doy. It's meant to be an int everywhere I just didn't update CollectionUpdateEvent. Good catch.